docs/README.md

```diff
@@ -0,0 +1,9 @@
+## Generating documentation locally
+
+Run `scripts/document.sh`
+
+## Update the documentation site:
+1. Clone [mapbox-navigation-ios](https://github.com/mapbox/mapbox-navigation-ios) to a mapbox-navigation-ios-docs folder alongside your MapboxDirections.swift clone, and check out the `gh-pages` branch.
```

Comment:
I'd rather keep the generated documentation out of the main github history, particularly if we move over to a separate mapbox.com subsite.